packages/next/client/link.tsx

```diff
@@ -45,7 +45,7 @@ function formatUrl(url: Url) {
   return url && typeof url === 'object' ? formatWithValidation(url) : url
 }
 
-type LinkProps = {
+export type LinkProps = {
   href: Url
   as?: Url | undefined
```

Comment:
Potentially stupid question, but isn't the ` | undefined` redundant here?

Reply:
It is.

Reply:
It doesn't hurt to have for explicitness.

Reply:
It's true that it doesn't hurt, but it's inconsistent with the lines below.
Clean and consistent code also gives me the peace of mind that the authors hold high standards over their code.

Reply:
Why is that related to my PR? ;)
Do you want me to remove this `undefined`?

Reply:
It's not related to your PR, it just caught my eye while I was reading through the changes.

Reply:
Cool, no problem to remove it, just a matter of decision :]